tests/ItemRow/ItemRow.spec.js

```diff
@@ -1,59 +1,16 @@
 import { React, shallow } from '../setup';
-import { ItemRow, Icon } from '../../lib';
-import Avatar from '../../lib/Avatar';
+import { ItemRow } from '../../lib';
 
 describe('Item Row', () => {
   let wrapper;
 
   beforeEach(() => {
-    wrapper = shallow(
-      <ItemRow
-        indicator="indicator"
-        label="label text"
-        participants={<Avatar />}
-      >
-        Hello world
-      </ItemRow>
-    );
+    wrapper = shallow(<ItemRow>Hello world</ItemRow>);
   });
 
   afterEach(() => {});
 
-  test('adds classes per prop', () => {
-    expect(wrapper.hasClass('has-indicator')).toEqual(true);
-    expect(wrapper.hasClass('has-label')).toEqual(true);
-  });
-
   test('renders children', () => {
-    expect(wrapper.find('.item-row__title').contains('Hello world')).toEqual(
-      true
-    );
-  });
-
-  test('renders an indicator', () => {
-    expect(wrapper.find('.item-row__indicator').contains('indicator')).toEqual(
-      true
-    );
-  });
-
-  test('renders an label', () => {
```

Comment:
Tests no longer required...everything is handled via the compound pattern and render children 👍